project/docker-entrypoint.sh

```diff
@@ -1,4 +1,7 @@
 python -W ignore manage.py collectstatic -c --noinput
 python -W ignore manage.py migrate --noinput
 python -W ignore manage.py creatersakey
-uwsgi --ini djam.ini & python manage.py rundramatiq
+echo "running dramatiq"
+nohup python manage.py rundramatiq &
+echo "running django"
+python manage.py runserver 0.0.0.0:8000
```

Comment:
are you sure we want to run it with the development server? You're proposing guvicorn as a dependency, why don't we use it here?

Reply:
it was a test due the static files issue we saw. So gunicorn can be removed. Is meant to be a `development` kind of docker.
We can integrate it and make it work again with gunicorn and nginx